package.json

```diff
@@ -212,6 +213,7 @@
     "loader-utils": "2.0.0",
     "memfs": "3.2.2",
     "node-fetch": "2.6.7",
+    "node-sass": "7.0.1",
```

Comment:
hmm so we are using dart-sass `"sass": "1.49.11",` can we remove this ?

you'll probably need to adjust api in that test https://github.com/sass/dart-sass

Reply:
Replaced by `dart-sass`